config.json

```diff
@@ -3,7 +3,7 @@
     "onBuildSuccess": null
   },
   "mc": {
-    "dev": true,
+    "dev": false,
```

Comment:
make sure dev isn't changed in the repo pls :)

Reply:
done